build.gradle

```diff
@@ -2,7 +2,7 @@ import edu.wpi.first.gradlerio.deploy.roborio.RoboRIO
 
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2022.2.1"
+    id "edu.wpi.first.GradleRIO" version "2022.1.1"

```

Comment:
We shouldn't be moving back versions. Can you check your local WPILib install is up to date?

Reply:
Resolved to 2022.3.1